src/Microsoft.DotNet.Wpf/src/PresentationCore/System/Windows/dataobject.cs

```diff
@@ -2732,20 +2732,19 @@ private object GetDataFromHGLOBAL(string format, IntPtr hglobal)
                     {
                         data = ReadStringFromHandle(hglobal, false);
                     }
-                    else if (IsFormatEqual(format, DataFormats.UnicodeText) ||
-                             IsFormatEqual(format, DataFormats.ApplicationTrust))
+                    else if (IsFormatEqual(format, DataFormats.UnicodeText))
```

Comment:
why we are removing `IsFormatEqual(format, DataFormats.ApplicationTrust))` check?

Reply:
Because it was specific to CAS scenarios on .NET Framework. It has also been removed in WinForms.

Reply:
Although not directly related to the code change, is the idea that objects from low-trust .NET Framework applications can be transferred to full-trust .NET Core applications?

Reply:
@miloush I'd have to dig deeper to give you a full answer, but you can see one of the types involved in the reference source for WPF.

https://referencesource.microsoft.com/#PresentationCore/Core/CSharp/MS/Internal/[ConstrainedDataObject.cs](https://referencesource.microsoft.com/#PresentationCore/Core/CSharp/MS/Internal/ConstrainedDataObject.cs)

Reply:
that's what caused my prompt, since this will no longer be used for low-trust FW objects...

https://referencesource.microsoft.com/#PresentationCore/Core/CSharp/System/Windows/Clipboard.cs,810